xcp_d/interfaces/prepostcleaning.py

```diff
@@ -276,10 +278,15 @@ def _run_interface(self, runtime):
         # Read in custom confounds file (if any) and bold file to be censored
         bold_file_uncensored = nb.load(self.inputs.in_file).get_fdata()
         if self.inputs.custom_confounds:
-            custom_confounds_tsv_uncensored = pd.read_csv(
-                self.inputs.custom_confounds,
-                header=None,
-            )
+            if os.path.exists(self.inputs.custom_confounds):
+                custom_confounds_tsv_uncensored = pd.read_csv(
```

Comment:
```suggestion
                custom_confounds_tsv_uncensored = pd.read_table(
```
Since it should be tab-delimited.